release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -2182,3 +2182,109 @@ xref:ocp-311-technology-preview[Technology Preview] and now generally available.
 To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest
 release, see xref:../upgrading/index.adoc#install-config-upgrading-index[Upgrade
 methods and strategies] for instructions.
+
+[[ocp-3-11-98]]
+=== RHBA-2019:0636 - {product-title} 3.11.98 Bug Fix and Enhancement Update
+
+Issued: 2019-04-03
+
+{product-title} release 3.11.98 is now available. The list of packages and
+bug fixes included in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2019:0636[RHBA-2019:0636] advisory.
+The container images included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2019:0637[RHBA-2019:0637] advisory.
+
+Space precluded documenting all of the bug fixes and enhancements for this
+release in the advisory. See the following sections for notes on upgrading and
+details on the bug fixes and enhancements included in this release.
+
+[[ocp-3-11-98-bug-fixes]]
+==== Bug Fixes
+
+* Administrative users were not able to access the cluster endpoints because permissions were defined incorrectly. Now, the correct permissions have been defined, and administrative users can use the `_cat` endpoints.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1548640[*BZ#1548640*])
+
+* Image garbage collection failed to remove an image correctly if it has only one tag but more than one repository associated with the image. This has now been resolved and garbage collection completes successfully.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1647348[*BZ#1647348*])
+
+* The `docker` registry Health Check would fail if the bucket was empty on AWS S3 environments, returning a `PathNotFound` message. Now, `PathNotFound` is treated a success and Health Check works as expected for empty buckets.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1655641[*BZ#1655641*])
```

Comment:
as a success?

Reply:
Fixed! Yeah that's a weird one... I thought about "treated as successful", but it really does mean "Health Check Success", so I went with your suggestion. Thanks!